lib/fluent/plugin/buf_file.rb

```diff
@@ -56,6 +51,14 @@ def initialize
       def configure(conf)
         super
 
+        unless @path
```

Comment:
In the future, should we validate `@path` value for avoding conflict trouble?
Fixed path should not allowed with multiprocess configuration or
support `%{worker_id}`/`%{plugin_id}` placeholder?